.circleci/config.yml

```diff
@@ -10,7 +10,7 @@ jobs:
       ignore:
         - gh-pages
     docker:
-      - image: googleapis/nox:0.17.0
+      - image: googleapis/nox:latest
```

Comment:
Is this relevant to the sample app?

Reply:
This was to fix a build error I was getting regarding the image not having `pip` in the virtual env. Upgrading to `latest` fixes the error. I will update the title of the PR to include this.